impl/src/test/java/org/ehcache/impl/internal/spi/copy/DefaultCopyProviderTest.java

```diff
@@ -84,6 +90,39 @@ public void testCreateValueCopierWithSerializer() {
     assertThat(copyProvider.createValueCopier(Long.class, mock(Serializer.class), config), instanceOf(SerializingCopier.class));
   }
 
+  @Test
```

Comment:
There is a generic test case for that in `ClassInstanceProviderTest#testReleaseCloseableInstance` which tests `Closeable#close` on instantiated ones